ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/AbstractPostProcessor.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
```

Comment:
You can update the copyright: 2019 -> 2020!

Reply:
ok